sdks/java/harness/src/test/java/org/apache/beam/fn/harness/debug/OutputSamplerTest.java

```diff
@@ -139,6 +139,69 @@ public void testActsLikeCircularBuffer() throws Exception {
     assertThat(samples, containsInAnyOrder(expected.toArray()));
   }
 
+  /**
+   * Test that elements with exceptions can be sampled. TODO: test that the exception metadata is
+   * set.
+   *
+   * @throws Exception when encoding fails (shouldn't happen).
+   */
+  @Test
+  public void testCanSampleExceptions() throws Exception {
```

Comment:
Instead of throws Exception, could we change this to IOException as required by `org.apache.beam.fn.harness.debug.OutputSampler.samples() throws IOException`? I realize all the other tests here do throws Exception. I thought it might be clearer to see the less generic Exception. Feel free to resolve this comment if you disagree.

Reply:
Done